rstudio.tera

```diff
@@ -0,0 +1,812 @@
+/* language constants like TRUE and  FALSE */
+.ace_constant.ace_language {
+  color: var(--blue);
+
+}
+
+.ace_constant.ace_numeric {
+  color: var(--mauve);
+
+}
+
+/* probably referring to escape characters, but not entirely clear */
+.ace_constant.ace_character.ace_escape {
+  color: var(--yellow);
+}
+
+.ace_constant.ace_other {
+  color: var(--subtext0);
+
+}
+
+/* Changes the color and style of code blocks in RMarkdown documents. */
+.ace_support.ace_function {
+  color: var(--sapphire);
+
+}
+
+.ace_support.ace_constant {
+  color: var(--teal);
+
+}
+
+.ace_support.ace_class {
+  color: var(--teal);
+
+}
+
+.ace_support.ace_type {
+  color: var(--teal);
+
+}
+
+.ace_entity.ace_name.ace_tag,
+.ace_meta.ace_tag,
+.ace_storage,
+.ace_storage.ace_type {
+  color: var(--blue);
+
+}
+
+.ace_invalid.ace_illegal {
+  color: var(--subtext0);
+  background-color: var(--red);
+}
+
+.ace_invalid.ace_deprecated {
+  color: var(--subtext0);
+  background-color: var(--yellow);
+}
+
+.ace_string {
+  color: var(--green);
+}
+
+.ace_string.ace_regexp {
+  color: var(--yellow);
+}
+
+.ace_comment {
+  color: var(--surface2);
+
+}
+
+/* could not find documentation, but looks very high-level. unclear what it actually is */
+.ace_variable {
+  color: var(--sapphire);
+}
+
+/* could not find documentation, but looks very high-level. unclear what it actually is */
+.ace_variable.ace_language {
+  color: var(--blue);
+}
+
+.ace_variable.ace_parameter {
+  color: var(--subtext0);
+
+}
+
+.ace_entity.ace_other,
+.ace_entity.ace_other.ace_attribute-name {
+  color: var(--teal);
+
+}
+
+.ace_entity.ace_name.ace_function {
+  color: var(--sapphire);
+}
+
+/* Characters that start a heading in RMarkdown documents. */
+.ace_markup.ace_heading {
+  color: var(--sapphire);
+}
+
+.nocolor.ace_editor .ace_line span {
+  color: var(--subtext0) !important;
+}
+
+.ace_bracket {
+  margin: 0 !important;
+  border: 0 !important;
+  background-color: rgba(128, 128, 128, 0.5);
+  color: var(--text);
+}
+
+.ace_marker-layer .ace_foreign_line {
+  position: absolute;
+  z-index: -1;
+  background-color: var(--base);
+}
+.ace_marker-layer .ace_active_debug_line {
+  position: absolute;
+  z-index: -1;
+  background-color: var(--surface1);
+}
+.ace_marker-layer .ace_find_line {
+  position: absolute;
+  z-index: -1;
+  background-color: var(--surface1);
+}
+
+.ace_console_error {
+  background-color: var(--surface1);
+  color: var(--red) !important;
+}
+
+#rstudio_console_output .ace_constant.ace_language {
+  color: var(--red) !important;
+}
+
+.terminal {
+  background-color: var(--base);
+  color: var(--subtext0);
+  font-feature-settings: "liga" 0;
+  position: relative;
+  user-select: none;
+  -ms-user-select: none;
+  -webkit-user-select: none;
+}
+
+/* Hic sunt dracones: unclear what happens after here and why we have hard-coded colors */
+
+.terminal.xterm-cursor-style-block.focus:not(.xterm-cursor-blink-on) .terminal-cursor {
+  background-color: #CCC;
+  color: #1e1e1e;
```

Comment:
You should be able to template `color` here right?

Reply:
See the above comment. Not sure where this is or why we have it hard-coded, that would maybe be a fix for down the road.